int32 x
int32 y
int32 tagId
int32 cameraId
